api/types/database.go

```diff
@@ -690,7 +691,7 @@ var (
     {
       "Effect": "Allow",
       "Action": "rds-db:connect",
-      "Resource": "arn:aws:rds-db:%v:%v:dbuser:%v/*"
+      "Resource": "arn:%v:rds-db:%v:%v:dbuser:%v/*"
```

Comment:
At what point would using `text/template` with named fields be safer than a sprint template with lots of `%v` verbs?

Reply:
yeah the point is probably now, updated at least these two templates to use `text/template`